src/client/at/game/Player.js

```diff
@@ -222,8 +229,26 @@ Player.prototype.registerAnimalRescued = function() {
   this._animalsRescued++;
 }
 
+Player.prototype.enemyKilled = function(type) {
+  // Different points based on enemy type?
+  this._enemiesKilled++;
+}
+
 Player.prototype.getScore = function() {
   return (this._carrotsCollected * this._carrotMultiplier) +
     (this._animalsRescued * this._animalMultiplier) +
     (this._enemiesKilled * this._enemyMultiplier);
 }
+
+Player.prototype.getStats = function() {
+  return {
```

Comment:
This is all the stuff that we are tracking stats on? Does anything else need to be added?